openedx_events/learning/signals.py

```diff
@@ -196,3 +197,39 @@
         "notification_data": UserNotificationData,
     }
 )
+
+# .. event_type: org.openedx.learning.thread.created.v1
+# .. event_name: FORUM_THREAD_CREATED
+# .. event_description: Emitted when a new thread is created in a discussion
+# .. event_data: DiscussionThreadData
+# Warning: This event is currently incompatible with the event bus, list/dict cannot be serialized yet
```

Comment:
could you move this warning to the event description? Like [this](https://github.com/openedx/openedx-events/blob/main/openedx_events/learning/signals.py#L154-L155). Same comment for the others